core-bundle/src/EventListener/DataContainer/PageUrlListener.php

```diff
@@ -211,9 +211,6 @@ private function recursiveValidatePages(int $pid, PageModel $rootPage): void
         }
     }
 
-    /**
-     * @throws DuplicateAliasException
-     */
```

Comment:
This is a good example IMO that shows why we should not exclude private methods from the `@throws` rule. In all other methods of this class we need to try/catch this exception (as we already do) and the `@throws` annotation would help us not to forget that.

Reply:
Agreed. 👍

Reply:
Added in 4b170582f71bd8906902e2b5fd5ef2205966ae50 again.